Makefile

```diff
@@ -20,15 +20,12 @@ pull:
 	echo $(APPS) | cut -d/ -f3 | xargs -P8 -n1 ./bin/update-git-repo.sh
 
 test:
-	# Linting
-	bundle exec rubocop . --parallel
```

Comment:
Shouldn’t we still keep this if we’re going to at least have some parts still written in Ruby?

Reply:
Hmm, I've read the commit message.

Jenkins does `bundle exec govuk-lint-ruby --parallel --format html --format clang app lib spec test` and I'm concerned that if we start to move Ruby files out of those folders we might lose linting without realising. Other than just remembering that and changing [`rubyLintDirs`](https://github.com/alphagov/govuk-jenkinslib/blob/5fe7a435cc54f9f162d8167016d5238139977c4c/vars/govuk.groovy#L163) as necessary I'm not sure how we could prevent that from happening though.

Reply:
If we think we'll retain the standard `lib` and `spec` dirs then I think it's fine.